test/service/test_app_logging.py

```diff
@@ -21,62 +25,114 @@
-                    with self.assertLogs(logger=log, level=level) as logs:
-                        requests.get(str(url), headers=headers)
-                    logs = [(r.levelno, r.getMessage()) for r in logs.records]
-                    headers = {
-                        'host': url.netloc,
-                        'user-agent': 'python-requests/2.32.2',
-                        'accept-encoding': 'gzip, deflate, br',
-                        'accept': '*/*',
-                        'connection': 'keep-alive',
-                        **headers,
-                    }
-                    self.assertEqual(logs, [
-                        (
-                            INFO,
-                            f"Received GET request for '/health/basic', "
-                            f"with {json.dumps({'query': None, 'headers': headers})}."),
-                        (
-                            INFO,
-                            "Authenticated request as OAuth2(access_token='foo_token')"
-                            if authenticated else
-                            'Did not authenticate request.'
-                        ),
-                        (
-                            level,
-                            'Returning 200 response. To log headers and body, set AZUL_DEBUG to 1.'
-                            if level == INFO else
-                            'Returning 200 response with headers {"Access-Control-Allow-Origin": '
-                            '"*", "Access-Control-Allow-Headers": '
-                            '"Authorization,Content-Type,X-Amz-Date,X-Amz-Security-Token,X-Api-Key", '
-                            f'"Content-Security-Policy": "default-src {sq("self")}", '
-                            '"Referrer-Policy": "strict-origin-when-cross-origin", '
-                            '"Strict-Transport-Security": "max-age=63072000; includeSubDomains; preload", '
-                            '"X-Content-Type-Options": "nosniff", '
-                            '"X-Frame-Options": "DENY", '
-                            '"X-XSS-Protection": "1; mode=block", '
-                            '"Cache-Control": "no-store"}. '
-                            'See next line for the first 1024 characters of the body.\n'
-                            '{"up": true}'
-                        )
-                    ])
+                    if reques_body:
+                        reques_body = json.dumps({'filters': json.dumps({'organ': {'is': ['foo']}})})
+                        headers = {
+                            'content-length': str(len(reques_body)),
+                            'content-type': 'application/json',
+                            **headers,
+                        }
+                    with self.subTest(level=level,
+                                      authenticated=authenticated,
+                                      request_body=reques_body,
+                                      azul_debug=azul_debug):
```

Comment:
```suggestion
                    with self.subTest(authenticated=authenticated,
                                      request_body=reques_body,
                                      azul_debug=azul_debug):
```

because `level` is derived from `azul_debug` you don't need to label the subtest with it.